geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloRunner.scala

```diff
@@ -107,9 +114,33 @@ object AccumuloRunner extends Runner {
             "change the Zookeeper timeout by setting the system property 'instance.zookeeper.timeout'.")
 
           import scala.concurrent.duration._
-          Await.result(Future(HdfsZooInstance.getInstance().getInstanceName)(ExecutionContext.global),  lookupTime.millis)
+          lazy val ff: Either[Throwable, String] = try {
+            // This call not only provides the instance name but in situations where the instance
+            // name is provided, this allows us to check that we have access to Accumulo dependencies.
+            // This helps prevent errors later and allows us to provide more relevant error messages now.
+            val i = HdfsZooInstance.getInstance().getInstanceName
+            Right(i)
+          } catch {
+            case e: Throwable => Left(e)
+          }
+          val future = Await.result(Future(ff)(ExecutionContext.global),  lookupTime.millis)
+
+          future match {
+            case Right(instanceId) =>
+              if (p.instance == null) p.instance = instanceId
+              else if (instanceId != p.instance) Command.user.warn("The provided instance ID (--instance) does not " +
+                "match the instance ID of the resolved Accumulo instance.")
+            case Left(e) => throw e
```

Comment:
Preface: this is from an outdated diff. 

As it is now: All of the errors I experienced while testing in my previous [comment](https://github.com/locationtech/geomesa/pull/1284#discussion_r100390347) are listed there.

Line 133: the error thrown here from the future has to do with looking up the instance name in zookeeper. If we're to this point then -i was not provided and getting the instance from the zookeeper was our last resort so if it's not found and error should be thrown. In addition we're only here if the zookeeper has been resolved, either by -z or the accumulo-site.xml (p.zookeepers != null)

The accumulo-site.xml is only needed with -z is not provided.